variables.tf

```diff
@@ -94,16 +94,8 @@ variable http_version {
 }
 
 variable minimum_protocol_version {
-  description = <<EOF
-    The minimum version of the SSL protocol that you want CloudFront to use for HTTPS connections. 
-    One of SSLv3, TLSv1, TLSv1_2016, TLSv1.1_2016 or TLSv1.2_2018. Default: TLSv1. 
-    NOTE: If you are using a custom certificate (specified with acm_certificate_arn or iam_certificate_id), 
-    and have specified sni-only in ssl_support_method, TLSv1 or later must be specified. 
-    If you have specified vip in ssl_support_method, only SSLv3 or TLSv1 can be specified. 
-    If you have specified cloudfront_default_certificate, TLSv1 must be specified.
-    EOF
-
-  type = string
+  description = "The minimum version of the SSL protocol that you want CloudFront to use for HTTPS connections. One of SSLv3, TLSv1, TLSv1_2016, TLSv1.1_2016 or TLSv1.2_2018. Default: TLSv1. NOTE: If you are using a custom certificate (specified with acm_certificate_arn or iam_certificate_id), and have specified sni-only in ssl_support_method, TLSv1 or later must be specified. If you have specified vip in ssl_support_method, only SSLv3 or TLSv1 can be specified. If you have specified cloudfront_default_certificate, TLSv1 must be specified."
```

Comment:
I suggest to adjust the linter so that heredoc syntax is accepted. IMO, it makes much more sense for such long descriptions.